config/eslint.config.js

```diff
@@ -49,6 +51,16 @@ module.exports = tseslint.config(
 		},
 		rules: {
 			...reactHooks.configs.recommended.rules,
+			// TODO-next-major: Elevate exhaustive-deps v as an error
+			// 'react-hooks/exhaustive-deps': 'error',
+			// TODO-next-major: Use the flat/recommended rules directly
```

Comment:
```suggestion
			// TODO-next-major: Use the flat/recommended rules directly and elevate react-query exhaustive deps rule to an error
```

Reply:
Nevermind, that's the default: https://github.com/TanStack/query/blob/main/packages/eslint-plugin-query/src/index.ts